src/modules/Dashboard/features/RespondentData/RespondentDataReview/ResponsesSummary/ResponsesSummary.hooks.ts

```diff
@@ -18,6 +20,8 @@ export const useResponsesSummary = ({
     DateFormats.MonthDayYearTimeSeconds,
   );
 
+  const respondent = useRespondentLabel({ hideLabel: true, sourceSubject });
```

Comment:
I think the `useRespondentLabel` function is doing too many things and we could probably get by with just calling `getRespondentName` directly here
```suggestion
  const respondent = sourceSubject?.secretUserId
    ? getRespondentName(sourceSubject.secretUserId, sourceSubject.nickname)
    : '';
```

I talk more about this in a subsequent comment